source/crud/crud.rst

```diff
@@ -1813,6 +1818,7 @@ Q: Where is ``singleBatch`` in FindOptions?
 Changes
 =======
 
+* 2020-01-10: Error if hint specified for unacknowledged update using legacy opcode
```

Comment:
Per discussion, this should also error for an unacknowledged OP_MSG. A user could be setting this in an unacknowledged write with OP_MSG in a 3.6-4.2 server. Since the option is unsupported, the server would always error, but no error would be returned. (Also made a [comment in SPEC-1540|https://jira.mongodb.org/browse/SPEC-1540?focusedCommentId=2718903&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2718903])

Reply:
I'm happy to make that change here, but I'll create a separate SPEC ticket to update the other, pre-existing options (e.g. `collation`) and let it be triaged accordingly at the next leads meeting.